CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* None.
+* Add optional AppError in app::Response  
```

Comment:
Changing the body to an optional is a breaking change for the SDKs and needs to be documented as such.

Reply:
fixed